catalog-item.html

```diff
@@ -55,62 +62,88 @@
+                    </dl>
+                </div>
+                <h2 class="card-title">sKISIRS#4587</h2>
+                <p class="card-text">
+                    As the first hero of the Neoh Universe, collection has over 9,999 unique skins drawn from the different missions and challenges he faced throughout his life.
                 </p>
                 <dl class="catalog-card-item-filter">
-                    <dt>Clothing</dt>
-                    <dd>SNAKE</dd>
-                    <dt>Eyes</dt>
-                    <dd>MIX</dd>
-                    <dt>Face</dt>
-                    <dd>GREEN</dd>
-                    <dt>Skin</dt>
-                    <dd>GREEN</dd>
-                    <dt>Background</dt>
-                    <dd>GREEN LIGHT</dd>
-                    <dt>Special</dt>
-                    <dd>CAT</dd>
+                    <div class="catalog-card-item-wrapper">
+                        <dt class="catalog-card-item-option">Clothing</dt>
+                        <dd class="catalog-card-item-meaning">SNAKE</dd>
+                    </div>
+                    <div class="catalog-card-item-wrapper">
+                        <dt class="catalog-card-item-option">Eyes</dt>
+                        <dd class="catalog-card-item-meaning">MIX</dd>
+                    </div>
+                    <div class="catalog-card-item-wrapper">
+                        <dt class="catalog-card-item-option">Face</dt>
+                        <dd class="catalog-card-item-meaning">GREEN</dd>
+                    </div>
+                    <div class="catalog-card-item-wrapper">
+                        <dt class="catalog-card-item-option">Skin</dt>
+                        <dd class="catalog-card-item-meaning">GREEN</dd>
+                    </div>
+                    <div class="catalog-card-item-wrapper">
+                        <dt class="catalog-card-item-option">Background</dt>
+                        <dd class="catalog-card-item-meaning">GREEN LIGHT</dd>
+                    </div>
+                    <div class="catalog-card-item-wrapper">
+                        <dt class="catalog-card-item-option">Special</dt>
+                        <dd class="catalog-card-item-meaning">CAT</dd>
+                    </div>
                 </dl>
-                <p class="catalog-card-item-price">PRICE: 0.15 ETH</p>
-                <button class="catalog-card-item-btn" type="button">
+                <p class="catalog-card-item-price">PRICE: <span class="accent">0.15 ETH</span> </p>
+                <button class="catalog-card-buy-btn" type="button">
                     <img src="./img/" alt="buy_icon">
                     BUY ON OPENSEA
                 </button>
                 <ul class="catalog-card-item-socialmedia-list">
                     <li class="catalog-card-item-socialmedia-item">
-                        <a href="#">
+                        <a href="#" class="catalog-card-item-socialmedia-link">
```

Comment:
I think, it is too long class name (:
You can use shorten version. For example, `card-social-link` 
And try to short other classnames, it's helps you to style your elements.